packages/bot-web-ui/src/assets/sass/load-modal.scss

```diff
@@ -0,0 +1,290 @@
+    }
+    &__dragndrop {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        border: dashed 3px var(--border-normal);
+        border-radius: $BORDER_RADIUS;
+        padding: 50px;
+        height: 100%;
+    }
+    &__title {
+        margin-top: 24px;
+        font-size: var(--text-size-s);
+    }
+    &__desc {
+        margin-top: 25px;
+        font-size: var(--text-size-xs);
+    }
+    &__icon {
+        margin: 0 auto;
+    }
+    &__upload {
+        margin: 10px auto 0;
+        width: fit-content;
+    }
+    &__preview {
+        height: 100%;
+
+        &-title {
+            text-align: left;
+            font-size: var(--text-size-s);
+            font-weight: bold;
+            line-height: 1.43;
+        }
+        &--hidden {
+            display: none;
+        }
+        &-workspace {
+            height: 100%;
+            padding: 15px;
+            margin-top: 8px;
+            border-radius: $BORDER_RADIUS;
+            border: solid 1px var(--border-normal);
+            position: relative;
+
+            .injectionDiv {
+                pointer-events: none;
+            }
+        }
+        &-close {
+            background-image: radial-gradient(at right top, $color_white 60%, transparent 90%);
+            position: absolute;
+            padding: 25px;
+            border-bottom-left-radius: 50%;
+            right: 0;
+            top: 0;
+            z-index: 99;
+            cursor: pointer;
+        }
+    }
+    &__footer {
+        position: absolute;
+        bottom: 0;
+        right: 25px;
+    }
+}
+
+.gd {
+    &__container {
+        padding: 25px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: $modal_container_height;
+
+        > * {
+            margin: 0 auto;
+        }
+    }
+    &__text {
+        margin-top: 16px;
+        font-size: var(--text-size-s);
+        font-weight: bold;
+        line-height: 1.43;
+
+        &--disabled {
+            color: var(--text-disabled);
+        }
+    }
+    &__buttons {
+        margin-top: 16px;
+        display: flex;
+        justify-content: space-between;
+
+        > * {
+            margin: 0 10px;
+        }
+    }
+    &__connect {
+        width: fit-content;
+        margin-top: 16px;
+    }
+    &__icon {
+        &--active {
+            // TODO: [fix-dc-bundle] Fix import issue with Deriv Component stylesheets (app should take precedence, and not repeat)
+            --fill-color1: #ffcf45 !important;
+            --fill-color2: #1ca362 !important;
+            --fill-color3: #4686f5 !important;
+        }
+        &--disabled {
+            // TODO: [fix-dc-bundle] Fix import issue with Deriv Component stylesheets (app should take precedence, and not repeat)
+            --fill-color1: #{$color-grey-8} !important;
+            --fill-color2: #{$color_grey-9} !important;
+            --fill-color3: #{$color-grey-10} !important;
```

Comment:
Is this still applicable?

Reply:
Yes. Currently css from core override css from bot.